lisa/tools/startup.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePosixPath
+
+from lisa.executable import Tool
+from lisa.tools.echo import Echo
+
+
+class Startup(Tool):
```

Comment:
How about `StartConfiguration`? Startup is like to control "start" of a machine.